CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## 3.3.5 (2022-09-22)
+- X-Ray Daemon integration testing [PR #174](https://github.com/aws/aws-xray-daemon/pull/174)
+- Run the E2E CI test in a provided region [PR #176](https://github.com/aws/aws-xray-daemon/pull/176)
+- Test linux zip in China & US Gov regions [PR #178](https://github.com/aws/aws-xray-daemon/pull/178)
```

Comment:
We do not need to include changes that do not impact the behavior of the daemon itself